lib/fastlane/plugin/android_sdk_update/actions/android_sdk_update_action.rb

```diff
@@ -22,9 +29,13 @@ def self.run(params)
         sdk_version = params[:compile_sdk_version] || properties[:compile_sdk_version] || UI.user_error!('No compile sdk version defined.')
 
         # Determine SDK dir and the sdkmanager
-        sdk_path = File.realpath("../..", FastlaneCore::CommandExecutor.which("android"))
         sdk_manager = File.expand_path("tools/bin/sdkmanager", sdk_path)
         Actions.lane_context[SharedValues::ANDROID_SDK_DIR] = sdk_path
+        # add the platform-tools directory to the PATH so calls to `adb` work from other plugins
+        if !":#{ENV['PATH']}:".include? ":#{sdk_path}/platform-tools:"
```

Comment:
On Linux this solution impacts.

Just running Screengrab results in this failure:
```
$ fastlane screengrab
[12:47:05]: The `adb` command could not be found on your PATH
[12:47:05]: Please ensure that the Android SDK is installed and the platform-tools directory is present and on your PATH

[!] adb command not found
```

If we want install the android-sdk reliable, I don't think we can solve it without either linking the executables or permanently adding the path.